api/src/db-connection.ts

```diff
@@ -257,6 +261,12 @@ export class Connection {
         ...relationships,
       },
     };
+
+    if (isSymbol(entity)) {
+      (entity as Symbol).attributes.disambiguated_id = entity.attributes.mathml
```

Comment:
```suggestion
      entity.attributes.disambiguated_id = entity.attributes.mathml
```
The type guard in the conditional is doing this cast for you.

Reply:
Didn't realize that. The miracle of TS.

Reply:
Isn't it wondeful?
![](https://media.giphy.com/media/VbEC9WchxkiWTL5PFo/giphy.gif)